content/departments/engineering/design/goals/index.md

```diff
@@ -0,0 +1,67 @@
+# Design team goals
+
+We have team-level design team goals to help us align and prioritize our
+focus as a team over the following 3-6 months.
+
+Our team-level goals cascade from the overall company [product vision and
+strategy](../../index.md#product-vision-and-strategy), and each member of the design team
+uses these to define their own individual goals.
+
+## (Draft) Design team goals for the next 3-6 months
+
+### Urgent and important
+
+#### Theme: Win the top 100 organizations
+
+1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.
+   - Key Result: Eliminate all the UI/UX-related must-haves from the scale
+     audit.
+   - Key Result: Design as a function contributes to closing a deal for a
+     2XL customer.
+2. **Objective**: Get closer to our users.
+   - Key Result: Each member of the design team participates in at least 1
+     sales, AE, or CE session per week.
+   - Key Result: Each member of the design team has at least `${x}` hours of
+     user time per `${cycle}`.
+
+#### Theme: Drive expansion in existing customers
+
+1. **Objective**: Identify opportunities to improve expansion in 2XL customers.
+   - Key Result: We have identified specific opportunities to improve
+     expansion within at least two 2XL customers.
+   - Key Result: Specific projects have been
+     prioritized with reference to the opportunities we identified.
+   - Key Result: 10% MAU growth is observed in customers where projects are executed.
+
+### Non-urgent and important
+
+#### Theme: Levelling up the design team
+
+1. **Objective**: Identify and follow through on opportunities for professional
+   growth.
+   - Key Result: Identify and carry out a professional development activity
+     related to the function of design.
+   - Key Result: Identify and carry out a professional development activity
+     related to the function of engineering.
+2. **Objective**: Level up storytelling skills.
+   - Key Result: Each member of the design team shares a #progress announcement.
+   - Key Result: Company-wide engagement in design review increases.
```

Comment:
I feel as if this is more an Objective that the team should aim to accomplish, and one of the KRs could be that design team members share to #progress. Another potential KR (pulling from a design offsite) is using some form of media (video snippets, user interview quotes, etc.) in a #progress post or team discussion.

Reply:
I tried to capture this by adding: "that exhibit strong storytelling skills."